marathon/gitlab/master.yml

```diff
@@ -0,0 +1,169 @@
+- name: Deploy gitlab
+  marathon:
+    marathon_uri: "{{ marathon_uri }}"
+    group_id: gitlab/master
+    group_json:
+        dependencies:
+            - /gitlab/core
+        apps:
+        -
+            id: gitlab
+            cpus: 4
+            mem: 8192
+            disk: 0
+            instances: 1
+            container:
+              type: DOCKER
+              volumes:
+              - containerPath: "gitlab"
+                mode: "RW"
+                persistent:
+                 type: "root"
+                 size: 10000
+              - containerPath: "/home/git/data"
+                hostPath: "gitlab"
+                mode: "RW"
+              docker:
+                image: sameersbn/gitlab:9.0.0
+                network: BRIDGE
+                privileged: false
+                parameters: "{{ docker_log_driver_parameters }}"
+                forcePullImage: true
+                portMappings:
+                - containerPort: 80
+                  protocol: tcp
+                  servicePort: "{{ servicePorts.gitlab_www }}"
+                - containerPort: 22
+                  protocol: tcp
+                  servicePort: "{{ servicePorts.gitlab_ssh }}"
+            env:
```

Comment:
we really need all those envs?

please keep only the needed one, and rather point to the list of available and default